lua/neo-tree/utils.lua

```diff
@@ -352,7 +352,7 @@ M.open_file = function(state, path, open_cmd)
         vim.cmd("vsplit " .. escaped_path)
         vim.api.nvim_win_set_width(winid, width)
       else
-        vim.cmd(open_cmd .. " " .. escaped_path)
+        pcall(vim.cmd, open_cmd .. " " .. escaped_path)
       end
     end
     events.fire_event(events.FILE_OPENED, path)
```

Comment:
By swallowing the error above, this FILE_OPENED event will now fire even if the file was not actually opened, creating more errors.